TBS-EACPD/infobase
# Typescript in client

Sort of working off of #803

- [x] builds work
- [x] VSCode problems matches the webpack output
- [x] storybook works
- [x] ES features (import, spread, etc.) work
    - [ ] have not tested decorators which are a special babel opt-in

I included a few example modules that were converted in #803, I'll have nested PRs for other examples. The Diffs tend to get huge because `import "x.js"` has to become `import "x"`


## How to review this

- check that vscode still highlights stuff appropriately, in js and in TS files
- check that you can build stuff locally
- look at your network tab and make sure I haven't blown up any bundles (or does CI do this?)
